CMSgov/design-system
# [WNMGDS-503] Add ability to disable/enable robust gold focus styles

### Added
- Boolean variable `$ds-include-focus-styles` to set to activate the robust gold focus styles.
- Added USWDS focus style variables to support original focus implementation 
- By default the robust gold focus styles are disabled 

### Changed
- Changed the focus styles to use if/else statements to activate and deactivate the robust gold focus styles.

**See the PR where the robust gold focus styles were added.** https://github.com/CMSgov/design-system/pull/510


### How to test
- Run the site locally 
- Add the `$ds-include-focus-styles` variable in a stylesheet like `packages/design-system/src/styles/index.scss` and set to `true` 
- You should see the robust gold focus styles (see the button example screenshot) 

![Screen Shot 2020-06-18 at 4 36 44 PM](https://user-images.githubusercontent.com/1449852/85069343-f48cff00-b181-11ea-87b3-ab598f41ba4c.png)
